contrib/runners/action_chain_runner/action_chain_runner.py

```diff
@@ -506,6 +508,20 @@ def _render_publish_vars(action_node, action_parameters, execution_result,
     @staticmethod
     def _resolve_params(action_node, original_parameters, results, chain_vars, chain_context):
         # setup context with original parameters and the intermediate results.
+        chain_parent = chain_context.get('parent', {})
+        pack = chain_parent.get('pack')
+        user = chain_parent.get('user')
+
+        if pack and user:
+            config_loader = ContentPackConfigLoader(
```

Comment:
Maybe a good idea to add some log statements here so it's easier to know that config is used (and also the values and what config is being used).